react-client-app/src/components/Home.jsx

```diff
@@ -101,20 +101,34 @@ const Home = () => {
         // sort scan reports
         scanreports = scanreports.sort((b, a) => (a.id > b.id) ? 1 : ((b.id > a.id) ? -1 : 0))
         // create a list of unique datapartners and make a batch query to get their data
-        const dataPartnerObject = {}
+        const datasetObject = {};
         scanreports.map(scanreport => {
-            dataPartnerObject[scanreport.data_partner] = true
+            datasetObject[scanreport.parent_dataset] = true;
         })
-        const dataPartnerIds = chunkIds(Object.keys(dataPartnerObject))
-        const dataPartnerPromises = []
+        const datasetIds = Object.keys(datasetObject)
```

Comment:
; ?

Reply:
This is because it is currently passing each dataset id individually rather than creating lists of id's to pass to an endpoint, because the endpoint doesn't exist yet

Reply:
I meant the need of a semi-colon at the end of the line

Reply:
Oh. Semicolons are optional in js but they get added when you build the app. I was working in the built version first to test so I didn't have to re build then copied it over afterwards and forgot to remove the semicolon. I'll remove it for consistency

Reply:
yeh sure cheers, wasnt sure if it'd auto do it or it'd give some eslint error :)